lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -312,16 +313,19 @@ internal class EvaluatingCompiler(
         )
     }()
 
+    @Deprecated("ExprNode is deprecated. Please use PIG generated AST. ")
+    fun compile(exprNode: ExprNode): Expression = compile(exprNode.toAstStatement())
+
     /**
      * Compiles an [ExprNode] tree to an [Expression].
```

Comment:
```suggestion
     * Compiles a [PartiqlAst] tree to an [Expression].
```

Reply:
(Could already be fixed in the other PR)

Reply:
I think it is fixed in the other PR, https://github.com/partiql/partiql-lang-kotlin/pull/528/files#diff-141d6e1b17ce7a7bb13abcea1c75b9129b09c56c6ca86f7417487835a7d875deR278.